src/main/java/lotto/domain/LottoResult.java

```diff
@@ -0,0 +1,40 @@
+package lotto.domain;
+
+import java.util.EnumMap;
+import java.util.EnumSet;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public record LottoResult(Map<LottoRank, Integer> results) {
+    private static final int INIT_NUM = 0;
+    private static final int PLUS_NUM = 1;
+    private static final String NEW_LINE = "\n";
+    private static final String VALUE_UNIT = "개";
+
+    public static LottoResult determineWinnings(final Lottos lottos, final WinningLotto winningLotto) {
+        EnumMap<LottoRank, Integer> results = new EnumMap<>(LottoRank.class);
+        EnumSet.allOf(LottoRank.class).forEach(rank -> {
+            if (rank != LottoRank.NONE) {
+                results.put(rank, INIT_NUM);
+            }
+        });
```

Comment:
준형님! 이부분 이렇게 변경하는건 어떻게 생각하세요?? 저도 비슷한 고민이 있어서 아래와 같이 수정해보았습니다 ㅎㅎ
```suggestion
        Stream.of(LottoRank.values())
                .filter(rank-> rank != LottoRank.NONE)
                .forEach(rank -> results.put(rank,INIT_NUM));
```